swap_meet/clothing.py

```diff
@@ -1,2 +1,2 @@
 class Clothing:
-    pass
+    pass
```

Comment:
This might just be githib being over cautious, but do .py files require a new line at the end to be valid? On the diff I see this red icon with hover text on clothing.py, decor.py, and vendor.py but not on item.py which has several new lines in it.

![Screen Shot 2022-03-04 at 2 35 39 PM](https://user-images.githubusercontent.com/97124999/156851238-67f9d95b-0d34-446e-813a-b7438e6dc745.png)

Reply:
I don't think so. I think this is just because I had the solutions in there, accidentally committed them, and then deleted them. I removed the extra new lines.